docs/src/main/asciidoc/stork-reference.adoc

```diff
@@ -0,0 +1,290 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Stork Reference Guide
+
+include::./attributes.adoc[]
+
+This guide is the companion from the xref:stork-getting-started.adoc[Stork Getting Started Guide].
```

Comment:
If you change the file name, it will have to be changed here too.

Reply:
Done.